src/Error/ErrorTrap.php

```diff
@@ -200,6 +200,12 @@ public function renderer(): ErrorRendererInterface
      */
     public function logger(): ErrorLoggerInterface
     {
+        $oldConfig = $this->getConfig('errorLogger');
+        if ($oldConfig !== null) {
+            deprecationWarning('The `errorLogger` configuration key is deprecated. Use `logger` instead.');
+            $this->setConfig('logger', $oldConfig);
+        }
```

Comment:
Should this clear the old config so the warning it's triggered every time?

Reply:
Good idea!